example/fcn-xs/init_fcnxs.py

```diff
@@ -0,0 +1,86 @@
+# pylint: skip-file
+import mxnet as mx
+import numpy as np
+import sys
+import logging
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+# make a bilinear interpolation kernel, return a numpy.ndarray
+def upsample_filt(size):
+    factor = (size + 1) // 2
+    if size % 2 == 1:
+        center = factor - 1
+    else:
+        center = factor - 0.5
+    og = np.ogrid[:size, :size]
+    return (1 - abs(og[0] - center) / factor) * \
+           (1 - abs(og[1] - center) / factor)
+
+def init_from_vgg16(ctx, fcnxs_symbol, vgg16fc_args, vgg16fc_auxs):
+    fcnxs_args = vgg16fc_args.copy()
+    fcnxs_auxs = vgg16fc_auxs.copy()
+    for k,v in fcnxs_args.items():
+        if(v.context != ctx):
+            fcnxs_args[k] = mx.nd.zeros(v.shape, ctx)
+            v.copyto(fcnxs_args[k])
+    for k,v in fcnxs_auxs.items():
+        if(v.context != ctx):
+            fcnxs_auxs[k] = mx.nd.zeros(v.shape, ctx)
+            v.copyto(fcnxs_auxs[k])
+    data_shape=(1,3,500,500)
+    arg_names = fcnxs_symbol.list_arguments()
+    arg_shapes, _, _ = fcnxs_symbol.infer_shape(data=data_shape)
+    rest_params = dict([(x[0], mx.nd.zeros(x[1], ctx)) for x in zip(arg_names, arg_shapes)
+            if x[0] in ['score_weight', 'score_bias', 'score_pool4_weight', 'score_pool4_bias', \

```

Comment:
yes, it a trick here~  because the class label 0 is the  background in image segmentation(general setting), and it has many background in one image, so if we init those weight to zeros, then at the first time it tend to ouput 0, and the back error will be large, and it benefit for the training.
here is the discuss about the initialize, https://groups.google.com/forum/#!msg/caffe-users/bsGdmJQ67Ts/tyy0zq2Cl28J , there is some answer from the  author which implentating the caffe version of fcn-xs